recipes/blas_openblas/meta.yaml

```diff
@@ -0,0 +1,21 @@
+package:
+  name: blas_openblas
+  version: 1.0.0
+
+build:
+  number: 0
+  track_features:
+    - blas_openblas
+
+test:
+  commands:
+    - conda list blas_openblas
+
+about:
+  home: NA
+  license: NA
+  summary: Metapackage to enable the use of OpenBLAS in an environment.
```

Comment:
Yeah, I just hacked together an explanation so that I could get something up here. Not claiming it is accurate or thought out. Wording tweaks welcome here.

> I feel like we need a summary page of how this operates - and for the home here, link to that explanation.

Maybe we can add a Readme with recipe and add a link to the feedstock with that Readme.